lib/modules/home/utils/onboarding.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:andax/models/story.dart';
+import 'package:andax/store.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+Future<StoryInfo?> needsOnboarding() async {
```

Comment:
The function name implies a return type of `bool`, not `StoryInfo?` Better divide into 2 functions: `Future<bool> isOnboardingNeeded()` and `Future<StoryInfo> retrieveOnboardingStory()`

Reply:
I've renamed the function. They 2 pieces are small and always used together, splitting feels redundant.